### ISSUE TITLE
"Directory '1.18' does not contain JSON file" even though it shouldn't be looking for 1.18, but for Fabric.

### ISSUE THREAD

**Issue by MakeSureDudeDies** [rakion99/minecraftlauncher#14]
# Hello, rakion99.

I have a issue with Fabric Installer.

The installer version is 0.11.2 and it is the universal JAR. (not the .exe)
The loader version is 0.14.21 (It's on 1.18)

I have installed everything correctly and it has to be something with the Launcher.

This appears everytime upon adding Fabric with the installer and opening the Launcher (After it loads):

![screenshot](https://github.com/rakion99/minecraftlauncher/assets/84237951/6a1a3a80-aea0-450d-be4b-631457f39e1c)

When i change all the fabric files to 1.18 (which it is looking for) the Launcher just does not run and halts.

**Comment by rakion99** [comment]
did you play 1.18 first? maybe you dont have it installed yet

**Comment by MakeSureDudeDies** [comment]
> did you play 1.18 first? maybe you dont have it installed yet

yeah i figured it out, kinda goofed out but then it errored that it needs 1.18.2 and i didnt check later (even tho it should run 1.18 just fine)
